### PR TITLE
new approval bundle

### PR DESCRIPTION
**Pull request by KatjaGlassConsulting**
Please include the new approval bundle into the Kimai Marketplace.

### CODE REVIEW

File: _store/katjaglass-approval-bundle.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- User lockdown -> a submitted/approved week can no longer be edited (apart from admins) - there is one lockdown date per user (LockdownPerUserBundle)
+- Teamlead/Admin can approve or deny the week
+- Overview of approvals, missing approvals and the status
+- Mailing options to recall approval tasks if outstanding
+
+## Requirement
+
+- Requires Kamai 2, V1.16.10 or higher
+- MetaFields plugin
+- LockdownPerUser plugin ([GitHub](https://github.com/kevinpapst/LockdownPerUserBundle))
```
Comment (kevinpapst) [inline _store/katjaglass-approval-bundle.md:59]:
```suggestion
- [Custom-fields plugin]({% link _store/keleo-custom-fields-bundle.md %})
- [Lockdown per user settings]({% link _store/keleo-lockdown-per-user-bundle.md %})
```

```suggestion
- 
```

File: _store/katjaglass-approval-bundle.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  - plugin
+bundle:
+    name: "ApprovalBundle"
+    clone: "https://github.com/KatjaGlassConsulting/ApprovalBundle"
+    command: "kimai:bundle:approval:install"
+    versions:
+      - ["0.9.1", "1.16.10"]
+---
+
+A plugin for Kimai to approve timesheets of users on a weekly basis including APIs.
```
Comment (kevinpapst) [inline _store/katjaglass-approval-bundle.md:45]: What does `including APIs` mean?

File: _store/katjaglass-approval-bundle.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+title: Weekly timesheets Approval
+name: Weekly timesheets Approval
+intro: "This bundle is available to perform a weekly approval for timesheets."
+developer: katjaglass
+date: "2022-08-24 12:00:00 +0100"
+icon: fa-solid fa-person-circle-check
+price: "0"
+github: https://github.com/KatjaGlassConsulting/ApprovalBundle
+download: https://github.com/KatjaGlassConsulting/ApprovalBundle/archive/master.zip
+type: client
```
Comment (kevinpapst) [inline _store/katjaglass-approval-bundle.md:11]:
```suggestion
type: plugin
```

File: _store/katjaglass-approval-bundle.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+title: Weekly timesheets Approval
+name: Weekly timesheets Approval
```
Comment (kevinpapst) [inline _store/katjaglass-approval-bundle.md:3]: Is "weekly" important or can we rename that?

```suggestion
title: Approval workflow for timesheets
name: Approval workflow for timesheets
```

**Review by kevinpapst — CHANGES_REQUESTED** [review]
Thanks 😄 added some minor comments

File: _store/katjaglass-approval-bundle.md (hunk 2 above)
Comment (KatjaGlassConsulting) [reply _store/katjaglass-approval-bundle.md]: I removed the API part. There are also three APIs supporting some external approval steps. That's in the readme and also in the Kimai auto-API-documentation.

**Comment by KatjaGlassConsulting** [comment]
Thanks, I included your suggestions. :)

**Pull request merged**